app/models/payola/subscription.rb

```diff
@@ -9,7 +9,7 @@ class Subscription < ActiveRecord::Base
     validates_presence_of :email
     validates_presence_of :plan_id
     validates_presence_of :plan_type
-    validates_presence_of :stripe_token
+    #validates_presence_of :stripe_token
```

Comment:
Instead of deleting the validation, could you change it to a custom validation that checks for presence of `stripe_token` if the plan's price is > 0?